File: 0205.isomorphic-strings/isomorphic-strings.java
class Solution {
    public boolean isIsomorphic(String s, String t) {
        int[] hash1 = new int[256];
        int[] hash2 = new int[256];
        for (int i = 0; i < s.length(); i++)
        {
            if (hash1[s.charAt(i)] != hash2[t.charAt(i)]) return false;
            hash1[s.charAt(i)] = hash2[t.charAt(i)] = i+1;
        }
        return true;
    }
}

File: 0205.isomorphic-strings/isomorphic-strings.cs
public class Solution {
    public bool IsIsomorphic(string s, string t) {
        for (int i = 0; i < s.Length; i++)
            for (int j = i + 1; j < s.Length; j++)
                if ((s[i] == s[j] && t[i] != t[j]) || (s[i] != s[j] && t[i] == t[j]))
                    return false;
        return true;
    }
}